containers/kubernetes/reference-content/managing-load-balancer-ips.mdx

```diff
@@ -0,0 +1,204 @@
+
+
+On the example below we will:
+
+* Create a new Load Balancer with the IP reserved before by patching the tea-svc service.
+* Check the IP address was correctly set on this service and that this service is now a Load Balancer one.
+* Delete the `tea-svc` service (showing that the IP address is not deleted).
+* Create a new Load Balancer with the IP reserved beforehand, by patching the `coffee-svc` service.
+
+By doing the steps above, we show that we can use a reserved IP on Load Balancer creation and that we can “move” this IP from one service to one another.
+
+```sh
+# kubectl get svc
+NAME         TYPE        CLUSTER-IP     EXTERNAL-IP   PORT(S)   AGE
+coffee-svc   ClusterIP   10.32.102.89   <none>        80/TCP    9s
+kubernetes   ClusterIP   10.32.0.1      <none>        443/TCP   3m56s
+tea-svc      ClusterIP   10.32.57.52    <none>        80/TCP    9s
+
+# kubectl patch svc tea-svc --type merge --patch '{"spec":{"loadBalancerIP": "51.159.24.7","type":"Load Balancer"}}'
+service/tea-svc patched
+
+# kubectl get svc
+NAME         TYPE           CLUSTER-IP     EXTERNAL-IP   PORT(S)        AGE
+coffee-svc   ClusterIP      10.32.102.89   <none>        80/TCP         44s
+kubernetes   ClusterIP      10.32.0.1      <none>        443/TCP        4m31s
+tea-svc      Load Balancer   10.32.57.52    <pending>     80:32434/TCP   44s
+
+# kubectl get svc
+NAME         TYPE           CLUSTER-IP     EXTERNAL-IP   PORT(S)        AGE
+coffee-svc   ClusterIP      10.32.102.89   <none>        80/TCP         45s
+kubernetes   ClusterIP      10.32.0.1      <none>        443/TCP        4m32s
+tea-svc      Load Balancer   10.32.57.52    51.159.24.7   80:32434/TCP   45s
+
+# kubectl delete svc tea-svc
+service "tea-svc" deleted
+
+# kubectl patch svc coffee-svc --type merge --patch '{"spec":{"loadBalancerIP": "51.159.24.7","type":"Load Balancer"}}'
+service/coffee-svc patched
+
+# kubectl get svc
+NAME         TYPE           CLUSTER-IP     EXTERNAL-IP   PORT(S)        AGE
+coffee-svc   Load Balancer   10.32.102.89   <pending>     80:31094/TCP   100s
+kubernetes   ClusterIP      10.32.0.1      <none>        443/TCP        5m27s
+
+# kubectl get svc
+NAME         TYPE           CLUSTER-IP     EXTERNAL-IP   PORT(S)        AGE
+coffee-svc   Load Balancer   10.32.102.89   51.159.24.7   80:31094/TCP   103s
+kubernetes   ClusterIP      10.32.0.1      <none>        443/TCP        5m30s
+```
+
+As you can see, we have been able to keep the same IP on different Load Balancers. This allows for instance, to keep the same DNS configuration and move it from one Load Balancer Instance type to another. We have also seen that Kapsule manages the configuration of your Load Balancer for you. The cloud controller manager is in charge of managing the complete lifecycle of your Load Balancer.
+
+## Demonstrate how to move an IP address across Kubernetes services
```

Comment:
@RoRoJ I think this example is exactly the same as the one in the previous section. Should we remove one of them?

Reply:
Thanks, the previous one shouldn't have been there, copy/paste error. Fixed